backend/command/src/items/services/item.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable, InternalServerErrorException, Logger } from "@nestjs/common";
+import { CommandBus } from "@nestjs/cqrs";
+import { InsertItemCommand } from "../commands/insert-item.command";
+import { ItemCronJobHandler } from "../cron/items.cron";
+import { InsertItemDto } from "../interfaces/insert-item.dto";
+
+@Injectable()
+export class ItemsService{
+    private readonly logger = new Logger(ItemsService.name)
+    constructor(private commandBus: CommandBus, private itemCronJobHandler:ItemCronJobHandler) {}
+
+    async handleBulkInsert(bulkItemInsertDTO: InsertItemDto[]){
+        let count = 0;
+        try {
+          for (const itemInsertDTO of bulkItemInsertDTO) {
+            // Use userid 0 for admin inserts
+            await this.commandBus.execute(new InsertItemCommand(itemInsertDTO, 0));
```

Comment:
Can we allow to have a custom userId? When we create data for the live version it could be helpful to assign these to a user.

Do you think it's enough to have it here to update it "in source code"?

Reply:
Done. Can be provided, but if omitted defaults to 0